### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Alchemist1411**
Just a grammatical error

### CODE REVIEW

**Comment by Deepak-suresh14** [comment]
> @googlebot

I signed it!

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 for machine learning. It has a comprehensive, flexible ecosystem of
 [tools](https://www.tensorflow.org/resources/tools),
 [libraries](https://www.tensorflow.org/resources/libraries-extensions), and
-[community](https://www.tensorflow.org/community) resources that lets
+[community](https://www.tensorflow.org/community) resources that let
```
Comment (Deepak-suresh14) [inline README.md:16]: It should be 'that **let** researchers' instead of 'that **lets** researchers '

Comment (Deepak-suresh14) [inline README.md:16]: hh

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@
...
 researchers push the state-of-the-art in ML and developers easily build and
```
Comment (Deepak-suresh14) [inline README.md:17]: It should be 'resources that **let** **researchers**' instead of 'resources that **lets** **researchers**'

**Comment by gbaned** [comment]
We will not be encouraging one liner grammatical changes as this is expensive process, thank you for your interest.
CC @mihaimaruseac

**Pull request closed without merging**